tests/multi_net/ssl_get_peer_cert.py

```diff
@@ -0,0 +1,61 @@
+# Simple test creating an SSL connection and transferring some data
+
+try:
+    import socket
+    import ssl
+except ImportError:
+    print("SKIP")
+    raise SystemExit
+
+PORT = 8000
+
+
+# This self-signed key/cert pair is randomly generated and to be used for
+# testing/demonstration only.  You should always generate your own key/cert.
+
+# To generate a new self-signed key/cert pair with openssl do:
+# $ openssl req -x509 -newkey rsa:4096 -keyout rsa_key.pem -out rsa_cert.pem
+# -days 365 -nodes
+# In this case CN is: micropython.local
+#
+# Convert them to DER format:
+# $ openssl rsa -in rsa_key.pem -out rsa_key.der -outform DER
+# $ openssl x509 -in rsa_cert.pem -out rsa_cert.der -outform DER
+#
+# Then convert to hex format, eg using binascii.hexlify(data).
```

Comment:
Instead of having this text in each file that uses the certificates, I suggest putting this in `tests/README.md` (in a new section in that file), and then put a single-line comment in the tests like this:

    # These are test certificates.  See tests/README.md for details.